### PR TITLE
suthesh/setup-storybook

### PR DESCRIPTION
**Pull request by suthesan-deriv**
Changes:

-   setup storybook

## Type of change

-   [ ] Bug fix
-   [x] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,104 +1,114 @@
...
+    "translate:download": "crowdin download -b crowdin && crowdin download -b crowdin -l zh-CN",
+    "translate:extract": "cd scripts && node extract-translations.js && cd ..",
+    "translate:upload": "crowdin upload sources -b crowdin",
+    "deploy": "npm run build && gh-pages -d public -b master",
+    "deploy-dev": "npm run build && gh-pages -d public",
+    "deploy-branch": "PATH_PREFIX=/deriv-com/br/$branch_name npm run build && gh-pages -d public -e br/$branch_name",
+    "prepare": "husky install",
+    "pre-commit-lint": "lint-staged",
+    "storybook": "start-storybook -p 6006",
+    "build-storybook": "build-storybook -c .storybook -o public/storybook"
```
Comment (mustofa-binary) [inline package.json:105]:
```suggestion
    "build:storybook": "build-storybook -c .storybook -o public/storybook"
```

**Review by mustofa-binary — CHANGES_REQUESTED** [review]
please add npm run build story book in staging github action workflow

File: src/pages/contact_us/_offices.js [hunk 2]
```diff
@@ -1,8 +1,7 @@
 import React from 'react'
 import { graphql, useStaticQuery } from 'gatsby'
 import styled from 'styled-components'
-import { Text, LocalizedLinkText } from 'components/elements/typography'
-import { Header, QueryImage } from 'components/elements'
+import { Header, QueryImage, Text, LocalizedLinkText } from 'components/elements'
```
Comment (sean-binary) [inline src/pages/contact_us/_offices.js:4]: Was this the online place where the import needed to be updated after the splitting of the component into separate files?

**Comment by sean-binary** [comment]
Could you address the 1 code smell? 🙏

File: src/pages/contact_us/_offices.js (hunk 2 above)
Comment (suthesan-deriv) [reply src/pages/contact_us/_offices.js:4]: Yup...the others were already importing LocalizedLinkText from 'components/elements'

**Review by sean-binary — DISMISSED** [review]

File: .github/workflows/staging.yml [hunk 3]
```diff
@@ -75,6 +75,7 @@ jobs:
             - run: npm run format
             - run: npm run test
             - run: npm run build
+            - run: npm run build:storybook
```
Comment (mustofa-binary) [inline .github/workflows/staging.yml:78]: please add noindex for storybook pls and exclude it from sitemap and robots

**Review by mustofa-binary — CHANGES_REQUESTED** [review]

**Review by mustofa-binary — APPROVED** [review]

**Pull request merged**